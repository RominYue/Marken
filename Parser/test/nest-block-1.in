> First Order Quote
> > List in Quote
> > * Unordered List 1-1
> > * Unordered List 1-2
> > Paragraph
> > 1. Ordered List 1-1
> >    > Quote in List
> >    >     Code in Quote
> >    >     Code in Quote
> >    >     * Fake List in Code
> > 2. Ordered List 1-2
> >        Code in List
> >        Code in List