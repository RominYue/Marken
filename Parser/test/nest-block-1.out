<blockquote><p>First Order Quote</p>
<blockquote><p>List in Quote</p>
<ul><li>Unordered List 1-1</li>
<li><p>Unordered List 1-2
Paragraph</p></li></ul>
<ol><li><p>Ordered List 1-1</p>
<blockquote><p>Quote in List</p>
<pre><code>Code in Quote
Code in Quote
* Fake List in Code</code></pre></blockquote></li>
<li><p>Ordered List 1-2</p>
<pre><code>Code in List
Code in List</code></pre></li></ol></blockquote></blockquote>
